### PR TITLE
[DOCS-382] Spliting AD tag doc into docker and K8s section

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

This PR is the first of a series that is going to move content outside of the AD section into the corresponding Docker and Kubernetes section.

It also aims at better reflecting the set of variables available for the container Agent.

### Motivation
Make the logic of Autodiscovery more streamlined from a platform angle.

### Preview link

* [Kubnernetes Tag extractions](https://docs-staging.datadoghq.com/gus/tag-extraction/agent/kubernetes/tag/?tab=containerizedagent)
* [Docker Tag extractions](https://docs-staging.datadoghq.com/gus/tag-extraction/agent/docker/tag/?tab=containerizedagent)

### CODE REVIEW

**Comment by l0k0ms** [comment]
This PR needs to be merged first: https://github.com/DataDog/documentation/pull/7007 before merging this one

File: content/en/agent/docker/tag.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  tag: "Documentation"
+  text: "Using tags with Datadog"
+- link: "/agent/autodiscovery/management"
+  tag: "Documentation"
+  text: "Manage which Container to include in the Agent Autodiscovery"
+---
+
+The Datadog Agent can create and assign tags to all metrics, traces, and logs emitted by a container based on its labels or environment variables.
+
+If running the Agent as a binary on a host, configure your tag extractions with the [Agent](?tab=agent) tab instructions. If running the Agent as a container configure your tag extraction with the  [Containerized Agent](?tab=containerizedagent) tab instructions.
```
Comment (cswatt) [inline content/en/agent/docker/tag.md]:
```suggestion
If you are running the Agent as a binary on a host, configure your tag extractions with the [Agent](?tab=agent) tab instructions. If you are running the Agent as a container, configure your tag extraction with the  [Containerized Agent](?tab=containerizedagent) tab instructions.
```

File: content/en/agent/docker/tag.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+## Extract Labels as Tags
```
Comment (cswatt) [inline content/en/agent/docker/tag.md]:
```suggestion
## Extract labels as tags
```

File: content/en/agent/docker/tag.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Starting with Agent v6.0+, the Agent can collect labels for a given container and use them as tags to attach to all data emitted by this container.
+
+{{< tabs >}}
+{{% tab "Containerized Agent" %}}
+
+To extract a given docker label `<LABEL_NAME>` and transform it as a tag key `<TAG_KEY>` within Datadog, add the following environment variable to the Datadog Agent:
```
Comment (cswatt) [inline content/en/agent/docker/tag.md]:
```suggestion
To extract a given Docker label `<LABEL_NAME>` and transform it as a tag key `<TAG_KEY>` within Datadog, add the following environment variable to the Datadog Agent:
```

File: content/en/agent/docker/tag.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+For example, you could set up:
+
+```shell
+DD_DOCKER_LABELS_AS_TAGS='{"com.docker.compose.service":"service_name"}'
+```
+
+{{% /tab %}}
+{{% tab "Agent" %}}
+
+To extract a given docker label `<LABEL_NAME>` and transform it as a tag key `<TAG_KEY>` within Datadog, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
```
Comment (cswatt) [inline content/en/agent/docker/tag.md]:
```suggestion
To extract a given Docker label `<LABEL_NAME>` and transform it as a tag key `<TAG_KEY>` within Datadog, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
```

File: content/en/agent/docker/tag.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+```yaml
+docker_labels_as_tags:
+  com.docker.compose.service: service_name
+```
+
+[1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+{{% /tab %}}
+{{< /tabs >}}
+
+## Extract Environment Variables as Tags
```
Comment (cswatt) [inline content/en/agent/docker/tag.md]:
```suggestion
## Extract environment variables as tags
```

File: content/en/agent/kubernetes/tag.md [hunk 6]
```diff
@@ -1,32 +1,25 @@
...
   tag: "Documentation"
   text: "Manage which Container to include in the Agent Autodiscovery"
 ---
 
-The Datadog Agent can create and assign tags to all metrics, traces, and logs emitted by a container based on its labels or environment variables.
-If you are working with a Kubernetes environment, the Agent can create and assign tags to all metrics, traces, and logs emitted by a Pod, based on its labels or annotations.
+The Agent can create and assign tags to all metrics, traces, and logs emitted by a Pod, based on its labels or annotations.
 
-If running the Agent as a binary on a host, configure your tag extractions with the [Agent](?tab=agent) tab instructions. If running the Agent as a container configure your tag extraction with the  [Containerized Agent](?tab=containerizedagent) tab instructions.
+If running the Agent as a binary on a host, configure your tag extractions with the [Agent](?tab=agent) tab instructions. If running the Agent as a container in your kubernetes cluster configure your tag extraction with the  [Containerized Agent](?tab=containerizedagent) tab instructions.
```
Comment (cswatt) [inline content/en/agent/kubernetes/tag.md]:
```suggestion
If you are running the Agent as a binary on a host, configure your tag extractions with the [Agent](?tab=agent) tab instructions. If you are running the Agent as a container in your Kubernetes cluster, configure your tag extraction with the [Containerized Agent](?tab=containerizedagent) tab instructions.
```

File: content/en/agent/kubernetes/tag.md [hunk 7]
```diff
@@ -44,29 +37,17 @@ annotations:
...
+Starting with Agent v7.17+, the Agent can Autodiscover tags from Docker labels. This process allows the Agent to associate custom tags to all data emitted by a container, without [modifying the Agent `datadog.yaml` file][1].
 
 ```yaml
-kubernetes_node_labels_as_tags:
-  <NODE_LABEL>: <TAG_KEY>
+com.datadoghq.ad.tags: '["<TAG_KEY>:TAG_VALUE", "<TAG_KEY_1>:<TAG_VALUE_1>"]'
 ```
 
-For example, you could set up:
+## Node Labels as Tags
```
Comment (cswatt) [inline content/en/agent/kubernetes/tag.md]:
```suggestion
## Node labels as tags
```

File: content/en/agent/kubernetes/tag.md [hunk 8]
```diff
@@ -82,41 +63,31 @@ DD_KUBERNETES_NODE_LABELS_AS_TAGS='{"app":"kube_app"}'
...
-For Agent v6.8.0+, use the following environment variable configuration to add all pod labels as tags to your metrics. In this example, the tag names are prefixed by `<PREFIX>_`:
+[1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+{{% /tab %}}
+{{< /tabs >}}
 
-```yaml
-kubernetes_pod_labels_as_tags:
-  *: <PREFIX>_%%label%%
-```
+## Pod Labels as Tags
```
Comment (cswatt) [inline content/en/agent/kubernetes/tag.md]:
```suggestion
## Pod labels as tags
```

File: content/en/agent/kubernetes/tag.md [hunk 9]
```diff
@@ -141,130 +112,78 @@ DD_KUBERNETES_POD_LABELS_AS_TAGS='{"*":"<PREFIX>_%%label%%"}'
...
-```
+**Note**: Using this method may [increase the number of metrics][2] for your organization and impact your billing.
 
+[1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+[2]: /developers/metrics/
 {{% /tab %}}
 {{< /tabs >}}
 
-## Docker
+## Pod Annotations as Tags
```
Comment (cswatt) [inline content/en/agent/kubernetes/tag.md]:
```suggestion
## Pod annotations as tags
```

**Pull request merged**